crypto/evp/c_allc.c

```diff
@@ -190,11 +190,6 @@ void openssl_add_all_ciphers_int(void)
     EVP_add_cipher(EVP_aes_256_cbc_hmac_sha1());
     EVP_add_cipher(EVP_aes_128_cbc_hmac_sha256());
     EVP_add_cipher(EVP_aes_256_cbc_hmac_sha256());
-#ifndef OPENSSL_NO_SIV
-    EVP_add_cipher(EVP_aes_128_siv());
```

Comment:
Does this mean we can remove the SIV code near the end of crypto/evp/e_aes.c? Presumably if it is only ever used provider side, then this libcrypto side code is no longer needed?

Reply:
Looks like it.

Reply:
This cascades wildly.

Reply:
I suspect there's a lot of code in libcrypto that can never be reached, since we foist the real work off onto the providers.
For example, when I was doing #12233 I had a lot of "Use local IV storage in <foo>" commits that tweaked cipher/encrypt/decrypt/etc. operations in crypto/evp/e_foo.c.  I couldn't find a codepath that would actually execute that code, but shied away from attempting to remove it all at that time, since it was a bigger chunk than I wanted to bite off.

Reply:
Legacy support for engines is the most likely way to trigger these paths.  The EVP get/set meth calls would be the other.